include/grpcpp/impl/codegen/interceptor.h

```diff
@@ -109,7 +109,13 @@ class InterceptorBatchMethods {
   /// Returns a modifable ByteBuffer holding the serialized form of the message
   /// that is going to be sent. Valid for PRE_SEND_MESSAGE interceptions.
   /// A return value of nullptr indicates that this ByteBuffer is not valid.
-  virtual ByteBuffer* GetSendMessage() = 0;
+  virtual ByteBuffer* GetSerializedSendMessage() = 0;
+
+  /// Returns a non-modifiable pointer to the original non-serialized form of
```

Comment:
The "original non-serialized form of the message" needs better qualification (mostly wrt. to what original means).
It should also be made clear that even for non-nullptr returned values this can be different from the serialized value returned by GetSerializedSendMessage.

Keeping the returned values in sync between GetSerializedSendMessage and GetSendMessage might be problematic.

Reply:
True, the semantics are being updated in a subsequent PR https://github.com/grpc/grpc/pull/17630